lib/scss/inc/_variables.scss

```diff
@@ -56,16 +63,6 @@ $header-height: 90px;
 $pre-bg: #f8f8f8;
 
 
-// Headings
-$h1-font-size: ($font-size-base - 1) * 1.65;
```

Comment:
Once again, can we keep the heading font sizes as the way they are, in line with Jayshree's feedback on the previous commit?